nutrition_labels/grant_tagger.py

```diff
@@ -80,21 +81,19 @@ class GrantTagger:
 
     Methods
     -------
-    fit_transform(data, train_data_id)
-        Fit a vectorizer using the text data
     transform(data)
         Vectorize text data using the fitted vectorizer
-    split_data(X_vect, y)
+    split_data(data, train_data_id)
         Split the data into train and test sets
     fit(X, y)
-        Train the model
+        Fit a vectorizer using the text data, and train the model
```

Comment:
I would also agree with the previous text, the model being the combination of vectorizer and classifier